src/main/scala/org/broadinstitute/dsde/rawls/openam/RawlsOpenAmClient.scala

```diff
@@ -0,0 +1,46 @@
+package org.broadinstitute.dsde.rawls.openam
+
+import org.broadinstitute.dsde.rawls.openam.UserInfoResponseProtocol._
+import org.broadinstitute.dsde.vault.common.openam.OpenAMClient
+import org.broadinstitute.dsde.vault.common.openam.OpenAMResponse._
+import spray.client.pipelining._
+import spray.http.{MediaTypes, HttpEntity, Uri}
+import spray.httpx.SprayJsonSupport._
+
+import scala.concurrent.Future
+
+/**
+ * Wrapper/replacement for vault-common's OpenAMClient that has the features we want
+ * (Rawls config, email extraction, etc)
+ */
+object RawlsOpenAmClient {
+  val baseClient = OpenAMClient
+  val config = RawlsOpenAmConfig
```

Comment:
we should not be divining config here, all config should be passed in through constructors. This is another problem doing it with an object. We want our code to be explicit about what config values it needs.